packages/app/app/components/LyricsView/index.js

```diff
@@ -44,6 +70,63 @@ export class LyricsView extends React.Component {
       </div>
     );
   }
+  
+  //Addition of WildLeons
+  pdfToHTML () {
+	  //Creation of a PDF file
+	  let pdf = new jsPDF();
+	  
+	  //Margins for shaping
+	  let margins = {
+		  top: 10,
+		  left: 10
+	  };
+	  
+	  //Get the name of the song and its artist and add them at the beginnig of the file
+	  let name = this.props.track.name;
+	  let artist = this.props.track.artist;
+	  pdf.text(name + " - " + artist, margins.left, margins.top);
+	  
+	  //Retrieve the lyrics only from the props.lyrics like in renderLyrics()
+	  let lyrics = _.get(this.props.lyrics, 'lyricsSearchResults', '');
+	  lyrics = _.get(lyrics, 'type', '');
+	  
+	  //Add variables to store the lyrics that will be on a page
+	  //and count the number of lines already passed before going to a next page
+	  let lyricsOfThePage = "";
+	  let nbrLineBreak = 0;
+	  
+	  for(let i = 0; i < lyrics.length; i++){
```

Comment:
These pages can be generated much more cleanly e.g. by using this function: https://lodash.com/docs#chunk